Scalar.Common/Git/GitProcess.cs

```diff
@@ -624,6 +624,7 @@ public Result MultiPackIndexRepack(string gitObjectDirectory, string batchSize)
 
             if (gitObjectsDirectory != null)
             {
+                gitObjectsDirectory = gitObjectsDirectory.Replace(Path.DirectorySeparatorChar, '/');
```

Comment:
Is there not a helper method for this already?

Reply:
Yes, thanks! `Paths.ConvertPathToGitFormat()` and I found another place that can call it.